build/broccoli/transpile-to-es5.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+const babel = require('broccoli-babel-transpiler');
+
+/**
+ * Transpiles a tree of ES2015+ JavaScript files to ES5 with Babel.
+ */
+module.exports = function transpileToES5(inputNode, modules = false) {
```

Comment:
I feel like we want to externalize the babel helpers. Every file that uses class (which is a lot) is going to emit this.

```
function _classCallCheck(instance, Constructor) { if (!(instance instanceof Constructor)) { throw new TypeError("Cannot call a class as a function"); } }

function _possibleConstructorReturn(self, call) { if (!self) { throw new ReferenceError("this hasn't been initialised - super() hasn't been called"); } return call && (typeof call === "object" || typeof call === "function") ? call : self; }

function _inherits(subClass, superClass) { if (typeof superClass !== "function" && superClass !== null) { throw new TypeError("Super expression must either be null or a function, not " + typeof superClass); } subClass.prototype = Object.create(superClass && superClass.prototype, { constructor: { value: subClass, enumerable: false, writable: true, configurable: true } }); if (superClass) Object.setPrototypeOf ? Object.setPrototypeOf(subClass, superClass) : subClass.__proto__ = superClass; }
```